### PR TITLE
modernize-wml-pipeline

### PR DESCRIPTION
**Pull request by animeshsingh**
Base it on components.yaml, and expose the parameters

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1227)

### CODE REVIEW

File: samples/ibm-samples/watson/watson_train_serve_pipeline.py [hunk 1]
```diff
@@ -16,60 +16,65 @@
...
+                                      value='https://raw.githubusercontent.com/user/repository/branch/creds.ini'),
+    train_code=dsl.PipelineParam(name='train-code', value='tf-model.zip'),
+    execution_command=dsl.PipelineParam(name='execution-command', value='\'python3 convolutional_network.py --trainImagesFile ${DATA_DIR}/train-images-idx3-ubyte.gz --trainLabelsFile ${DATA_DIR}/train-labels-idx1-ubyte.gz --testImagesFile ${DATA_DIR}/t10k-images-idx3-ubyte.gz --testLabelsFile ${DATA_DIR}/t10k-labels-idx1-ubyte.gz --learningRate 0.001 --trainingIters 20000\''),
+    framework= dsl.PipelineParam(name='framework', value='tensorflow'),
+    framework_version = dsl.PipelineParam(name='framework-version', value='1.5'),
+    runtime = dsl.PipelineParam(name='runtime', value='python'),
+    runtime_version = dsl.PipelineParam(name='runtime-version', value='3.5'),
+    run_definition = dsl.PipelineParam(name='run-definition', value='wml-tensorflow-definition'),
+    run_name = dsl.PipelineParam(name='run-name', value='wml-tensorflow-run'),
+    model_name=dsl.PipelineParam(name='model-name', value='wml-tensorflow-mnist'),
```
Comment (Ark-kun) [inline samples/ibm-samples/watson/watson_train_serve_pipeline.py]: You can do this without explicit `PipelineParam`s:
```suggestion
    model_name='wml-tensorflow-mnist',
```

Comment (animeshsingh) [reply samples/ibm-samples/watson/watson_train_serve_pipeline.py]: thanks - fixed

**Comment by animeshsingh** [comment]
guess this can be merged @Ark-kun @vicaire

**Comment by Ark-kun** [comment]
> guess this can be merged @Ark-kun @vicaire

Sure.
It's just that usually the PR author approves the PR if he's the owner - making the final call.

**Pull request merged**